__tests__/app.test.js

```diff
@@ -94,27 +93,66 @@ describe("/api/articles/:article_id", () => {
   });
 });
 
-  describe("/api/articles", () => {
-    it("GET responds with status code 200", () => {
-      return request(app).get("/api/articles").expect(200);
-    });
-    it("GET responds with slug and description fields", () => {
-      return request(app)
-        .get("/api/articles")
-        .then((response) => {
-            expect(response.body.articles).toBeSortedBy("created_at", { descending: true });
-            expect(response.body.articles.length).toBe(13)
-            response.body.articles.forEach((article) => {
-            expect(article).toMatchObject({
-                author: expect.any(String),
-                title: expect.any(String),
-                article_id: expect.any(Number),
-                topic: expect.any(String),
-                created_at: expect.any(String),
-                votes: expect.any(Number),
-                article_img_url: expect.any(String),
-                comment_count: expect.any(Number)
-              });
+describe("/api/articles", () => {
+  it("GET responds with status code 200", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  it("GET responds with slug and description fields", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        expect(response.body.articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        expect(response.body.articles.length).toBe(13);
+        response.body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
         });
-    });
-  })});
\ No newline at end of file
+      });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  it("GET responds with status code 200", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments.length).toBe(2);
+        expect(response.body.comments[0].article_id).toBe(3);
+        expect(response.body.comments[0]).toMatchObject({
+          comment_id: expect.any(Number),
+          body: expect.any(String),
+          article_id: expect.any(Number),
+          author: expect.any(String),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
```

Comment:
You need to check the order of all the comments, i would recommend picking an article with more comments so you can be confident your sorting is working, use something like jest-sorted to check the order